packages/itwin/tree-widget/src/components/trees/imodel-content-tree/IModelContentTreeDefinition.ts

```diff
@@ -708,13 +670,13 @@ function getElementsSelectProps(props?: { modelClass?: string }) {
         supportsFiltering: true,
       },
     },
-    {
+  ];
+
+  if (!props?.elementClass) {
```

Comment:
This condition is based on the assumption that `getElementsSelectProps` will be called with `"BisCore.GeometricElement2d" | "BisCore.GeometricElement3d" | undefined`, but the function doesn't enforce this. For example, it won't work correctly if `props.elementClass` is one of these:
- `BisCore.Element`,
- `BisCore.GroupInformationElement`.

Reply:
Fixed